app/views/early_career_payments/claims/itt_year.html.erb

```diff
@@ -21,6 +22,16 @@
 
             <%= errors_tag current_claim.eligibility, :itt_academic_year %>
 
+            <% if qualification_symbol.in?([:undergraduate_itt, :postgraduate_itt]) %>
+              <div id="disciplinary_action-hint" class="govuk-hint">
+                <% if qualification_symbol == :undergraduate_itt %>
```

Comment:
`qualification_symbol == :undergraduate_itt ? "undergraduate" : "postgraduate"` merged into that sentence?